ci/test_python_common.sh

```diff
@@ -10,7 +10,8 @@ set -euo pipefail
 
 rapids-logger "Downloading artifacts from previous jobs"
 CPP_CHANNEL=$(rapids-download-conda-from-github cpp)
-PYTHON_CHANNEL=$(rapids-download-conda-from-github python)
+PYTHON_CHANNEL=$(rapids-download-from-github conda_python "$(rapids-package-name conda_python cudf)")
+PYTHON_NOARCH_CHANNEL=$(rapids-download-from-github conda_python "$(rapids-package-name cudf-noarch --pure)")
```

Comment:
It seems like `conda_python` still needs to be specified, is that right?
```suggestion
PYTHON_NOARCH_CHANNEL=$(rapids-download-from-github conda_python "$(rapids-package-name conda_python cudf-noarch --pure)")
```